### DISCUSSION TITLE
HELP_AND_FAQ_URL var has no effect

### DISCUSSION THREAD

**Discussion by m3xw3ll** [danny-avila/LibreChat#2281] [Troubleshooting] [answered]
### What happened?

Tried the different configurations for the FAQ button described [here](https://docs.librechat.ai/install/configuration/dotenv.html#ui) without any effect. If I add a custom URL in the `.env` file, it still opens https://librechat.ai/

### Steps to Reproduce

1. Run `vim .env`
2. Add `HELP_AND_FAQ_URL=http://www.google.de`
3. Run` docker compose restart`

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
I just tested it and it works. You likely have the variable configured twice in your .env file, which is likely if you copied the example, which sets it to https://librechat.ai

Do a quick CTRL+F in your .env file

**Reply by m3xw3ll** [reply]
Was working with an old version. After pulling it works, thanks!
